ci

```diff
@@ -84,4 +95,21 @@ case "${1:?}"-"${2:?}" in
         for f in `ls ${BuildDir}/_build/dev*/rel/epoch/log/epoch.log`; do echo \"\"; echo \"$f:\"; echo \"\"; cat $f; done
         for f in `ls ${BuildDir}/_build/dev*/rel/epoch/log/crash.log`; do echo \"\"; echo \"$f:\"; echo \"\"; cat $f; done
         ;;
+    deploy-package)
+        # Free Travis version does not support SSH key pairs, thus using encrypted files for SSH deploy key
+        # https://docs.travis-ci.com/user/encrypting-files/
+        openssl aes-256-cbc -K $encrypted_a2e547bcd39e_key -iv $encrypted_a2e547bcd39e_iv -in deployment/ansible/files/travis/deploy_key.enc -out /tmp/deploy_rsa -d
+        # ssh agent does not accept world readable keys
+        chmod 600 /tmp/deploy_rsa
+        eval "$(ssh-agent -s)"
+        ssh-add /tmp/deploy_rsa
+
+        BuildDir="${3:?}"
+        DeployEnv="${4:?}"
+        TARBALL="${BuildDir}/_build/prod/rel/epoch/epoch-${Version}.tar.gz"
```

Comment:
Let me double check this path.

Reply:
(Not necessary for this PR.)

Please do not use this package but rather `"${BuildDir:?}"/_build/prod/rel/epoch/epoch-"${Version:?}"-"${Prefix:?}".tar.gz` as it is what we upload.

If possible also please consider changing https://github.com/aeternity/epoch/pull/362/files#diff-35ea51baf1fe7f0142ad5f950855dde0R74 from `cp -p` to `mv` so to ensure only one copy of the package is around in all parts of the CI scripts.

Reply:
sure, here is it ccc3ca9

Reply:
Thanks. Interesting to check CI passes.